logstash-core/spec/logstash/runner_spec.rb

```diff
@@ -24,8 +24,8 @@ def run(args); end
       let(:args) { ["-e", ""] }
 
       it "should execute the agent" do
-        expect(subject.agent).to receive(:add_pipeline).once
-        expect(subject.agent).to receive(:execute).once
+        expect_any_instance_of(LogStash::Agent).to receive(:add_pipeline).once
+        expect_any_instance_of(LogStash::Agent).to receive(:execute).once
```

Comment:
This expectation was needed to change, if we setup the `Agent` in the `Runner#initialize` method the `node_name` value wasn't not available before the `#run` is executed.